File: packages/binary-gap-js/src/main.test.js
// Hey Emacs, this is -*- coding: utf-8 -*-
/* global describe it */

import assert from 'assert';

import { solution } from './main.js';

describe('Array', () => {
  describe('BinaryGap solution()', () => {
    it('should return 42', () => {
      assert.equal(solution(10), 42);
    });
  });
});

File: packages/sparse-binary-decomposition/cpp14/src/main.test.cpp
// Hey Emacs, this is -*- coding: utf-8 -*-

#include "main.hpp"

#define BOOST_TEST_MODULE BinaryGap
#include <boost/test/unit_test.hpp>

// NOLINTNEXTLINE
BOOST_AUTO_TEST_CASE(binary_gap) {
  BOOST_REQUIRE_EQUAL(18, solution(26));
  BOOST_REQUIRE_EQUAL(0, solution(0));
}
